docs/src/documentation-pages/dev/quickstart.md

```diff
@@ -7,27 +7,38 @@ This quickstart describes the initial setup required to run an instance of Cross
 
 ### Initial Setup
 
-1. Install [Node.js](https://nodejs.org/en/download/) 18 and [Docker Compose](https://docs.docker.com/compose/install/). Make sure the Docker daemon is running.
+1. Mac Users - before starting the initial setup, ensure you have already completed the following: [development environment for mac-based computers](https://github.com/cisagov/development-guide/blob/develop/dev_envs/mac-env-setup.md).
 
-2. Copy root `dev.env.example` file to a `.env` file.
+   Click on the link for the location of [Crossfeed Github](https://github.com/cisagov/crossfeed) account.
+
+2. Install [Node.js](https://nodejs.org/en/download/) 18 and [Docker Compose](https://docs.docker.com/compose/install/). Make sure the Docker daemon is running.
+<!--NOTE: For Docker make sure once downloaded:
+
+   ```bash
```

Comment:
Is this only for mac?

Reply:
Maybe move to "common issues"?

Reply:
Agreed

There is an entry there for Mac issues

Reply:
Moved to the common issues only